src/Rules/Library/BoundingRectangleCompletelyObscuresContainer.cs

```diff
@@ -34,16 +36,19 @@ public override bool PassesTest(IA11yElement e)
         protected override Condition CreateCondition()
         {
             // Windows and dialogs can be any size, regardless of their parents
+var isDialog = Pane & IsDialog;
```

Comment:
nit: it looks like L39 lost its indentation by accident

Reply:
Revised line:
```suggestion
            var isDialog = Pane & IsDialog;
```